GomokuClient/packages/gomoku-core/src/api/schema.json

```diff
@@ -436,6 +467,16 @@
         },
         "additionalProperties": false
       },
+      "PaginationMetadata": {
+        "required": ["hasMoreItems"],
+        "type": "object",
+        "properties": {
+          "hasMoreItems": {
```

Comment:
Including only hasMoreItems can be limiting. It tells the client if there are more items, but not how many items there are in total. This can hinder the user experience in certain scenarios:
- For instance, if you want to show “Page X of Y” or show progress, you’ll need a TotalCount.
- Adding TotalCount to the metadata can be beneficial for cases where the UI needs more information about the dataset (let's say i want to display and say, we have total 228 games player)

Reply:
Good point. Added total count